src/index.css

```diff
@@ -1,14 +1,20 @@
 @tailwind base;
 @tailwind components;
 
+:root {

```

Comment:
not very nice since you're defining the default theme colors here too.

don't know whats a cleaner way of doing it tho

Reply:
i think setting these values on app startup is too jank prob

Reply:
It has to be like that, nothing I can do about it

Reply:
yeah :(